torchbenchmark/models/opacus_cifar10/__init__.py

```diff
@@ -17,32 +18,29 @@ def __init__(self, device=None, jit=False):
         self.jit = jit
 
         self.model = models.resnet18(num_classes=10)
-        self.model = convert_batchnorm_modules(self.model)
+        self.model = ModuleValidator.fix(self.model)
         self.model = self.model.to(device)
 
         self.example_inputs = (
```

Comment:
Can we use a variable `batch_size` to replace `64` on line 25 and line 30?

Reply:
Yup